### ISSUE TITLE
File taking minutes to open, generating high load on browser

### ISSUE THREAD

**Issue by acsfer** [ONLYOFFICE/DocumentServer#1392]
**Do you want to request a *feature* or report a *bug*?**

Bug report

**What is the current behavior?**

When loading the attached file, either trough Nextcloud or at https://kim.teamlab.info/example/ it takes several minutes to open (up to 20min on some computers) and it generates high load (CPU at 100%, RAM > 2Gb) on the browser (tested on latest Chrome and Firefox).

On some PCs, it **crashed** the Browser.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**

Try to open the attached file on the Editor

**What is the expected behavior?**

File should open in seconds

**Did this work in previous versions of DocumentServer?**

Can't tell as it hasn't been tested.

**DocumentServer version:**

v6.3.2 Entreprise Edition

**Operating System:**

Windows 10

**Browser version:**

Brave Version 1.28.106 Chromium: 92.0.4515.159  (64 bits)

Problematic file: [Compil Natio LIR  LCR 2021 - 08 - 19 - Vierge.xlsx](https://github.com/ONLYOFFICE/DocumentServer/files/7032433/Compil.Natio.LIR.LCR.2021.-.08.-.19.-.Vierge.xlsx)

**Comment by ShockwaveNN** [comment]
I know I can only ask you, since we do not forbid access to kim.teamlab.info - but please do not use at for any kind of testing

Regarding this exact file -we'll take a look at this file

**Comment by acsfer** [comment]
Noted.

**Comment by ShockwaveNN** [comment]
Ah got it

**Comment by acsfer** [comment]
No problem. I just used it to isolate any problem with our own setup of DocumentServer regarding this specific file.

**Comment by ShockwaveNN** [comment]
@acsfer Yeah, we see that this file have 64MB sheet6 so I think this file is just to big for us, but we'll create issue in case our dev team can optimize it

**Comment by Rita-Bubnova** [comment]
Hello.
@acsfer, I can confirm - this is bug, issue 52134 in our private issue tracker.

**Comment by acsfer** [comment]
@Rita-Bubnova 👍

**Comment by acsfer** [comment]
Any news on this bug? We have countless (similar) files likes these we an't worn on outside Microsoft Office.

**Comment by Rita-Bubnova** [comment]
@acsfer, sorry, currently there is no news.
